pkg/process/monitor/process_monitor.go

```diff
@@ -186,10 +234,12 @@ func (pm *ProcessMonitor) mainEventLoop() {
 			}
 
 			pm.eventCount.Inc()
+			pm.tel.events.Add(1)
```

Comment:
why do we need both?

Reply:
eventCount got logged and reseted every 2 minutes, I merged and use the telemetery logic to log in the console